infra/modules/service/main.tf

```diff
@@ -79,7 +80,7 @@ resource "aws_ecs_task_definition" "app" {
       cpu                    = var.cpu,
       networkMode            = "awsvpc",
       essential              = true,
-      readonlyRootFilesystem = true,
+      readonlyRootFilesystem = var.enable_service_execution ? !var.enable_service_execution : true,
```

Comment:
I think this would be a little cleaner as just `readonlyRootFilesystem = !var.enable_service_execution` if possible?

Reply:
Good catch. I brought this over from a repo where this is a two way check (i.e. you can explicitly also say "turn off read only"). I'll fix this for now.